lib/sles4sap/sdaf_naming_conventions.pm

```diff
@@ -0,0 +1,298 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+#
+# Library used to generate simple strings according to SDAF naming conventions
+
+package sles4sap::sdaf_naming_conventions;
+
+use strict;
+use warnings;
+use testapi;
+use Exporter qw(import);
+use Carp qw(croak);
+use mmapi qw(get_current_job_id);
+
+=head1 SYNOPSIS
+
+Library contains functions that handle SDAF naming conventions used in various strings.
+Mostly used for generating resource  names, file names or file paths.
+Please try not to add here complex functions that do much beyond returning a string.
+
+=cut
+
+our @EXPORT = qw(
+  homedir
+  deployment_dir
+  log_dir
+  sdaf_scripts_dir
+  env_variable_file
+  get_sdaf_config_path
+  get_tfvars_path
+  generate_resource_group_name
+  convert_region_to_long
+  convert_region_to_short
+);
+
+
+# SDAF uses own internal 4 character abbreviations for PC region names
+# This is a matrix used for translation.  It contains only commonly used regions, if you need to extend the list
+# you can find definitions in the function B<get_region_code> located in sdaf shell script:
+# https://github.com/Azure/sap-automation/blob/3c5d0d882f5892ae2159e262062e29c2b3fe59d9/deploy/scripts/deploy_utils.sh#L403
```

Comment:
I feel like this comment can also be part of the POD. Sort of like:
```
=head2 %sdaf_region_matrix

B<SDAF> uses own internal 4 character abbreviations for Public Cloud region names.

This is an internal matrix used for translation.  It contains only commonly used regions,
if you need to extend the list you can find definitions in the function B<get_region_code>
located in sdaf shell script at:
L<https://github.com/Azure/sap-automation/blob/3c5d0d882f5892ae2159e262062e29c2b3fe59d9/deploy/scripts/deploy_utils.sh#L403>

=cut

```

Reply:
Oh, that is a very good idea. Thanks, haven't thought about this. 
Just pushed the changes.